platform/platform-app/src/main/resources/features.xml

```diff
@@ -776,6 +776,12 @@
         <bundle>mvn:ddf.platform.country/platform-country-converter-local/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-usng4j" install="manual" version="${project.version}"
```

Comment:
Unless we've overloaded this, you probably want this to be auto so that it starts automatically when you put it in the deploy directory